tests/css/CSSCombBearTest.py

```diff
@@ -0,0 +1,277 @@
+example {
+color: red;
+margin-bottom: 100px;
+margin-right: 150px;
+margin-left: 80px;
+background-color: lightblue}
+"""
+
+test_file4 = """
+example{
+color: red;
+margin-bottom: 100px;
+margin-right: 150px;
+margin-left: 80px}
+"""
+
+test_file5 = """
+example{
+    color: red;
+    margin-bottom: 100px;
+    margin-right: 150px;
+}
+"""
+
+test_file_always_semicolon = """
+example{
+    color:red
+}
+"""
+
+test_file_space_after = """
+example {
+    margin-bottom: 100px;
+}
+"""
+
+test_file_space_after_close_brace = """
+example {
+    margin-bottom: 100px;
+}
+"""
+
+test_file_remove_empty_rulesets = """
+example {
+    margin-bottom: 100px;
+}
+example2 {}
+"""
+
+test_file_add_semicolon = """
+example {
+    margin-bottom: 200px
+}
+"""
+
+test_file_color_case = """
+example {
+    color: #fff;
+}
+"""
+
+test_file_color_case2 = """
+example {
+    color: #FFF;
+}
+"""
+
+test_file_block_indent = """
+example {
+color: tomato;
+}
+"""
+
+test_file_block_indent2 = """
+example {
+    color:tomato;
+}
+"""
+
+test_file_color_shorthand = """
+example {
+    color: #fc0;
+    background: #ccc;
+}
+"""
+
+test_file_color_shorthand2 = """
+example {
+    color: #ffcc00;
+    background: #cccccc;
+}
+"""
+
+test_file_leading_zero = """
+.a {
+    padding: 0.1vh;
+}
+"""
+
+test_file_leading_zero2 = """
+.a {
+    padding: .1vh;
+}
+"""
+csscombconfig = os.path.join(os.path.dirname(__file__),
+                             "test_files",
+                             "csscomb.json")
+
+settings = {
+    "always-semicolon": "True",
+    "color-case": "upper",
+    "eof-newline": "True",
+    "leading-zero": "False",
+    "sort-order-fallback": "abc",
+    "space-before-colon": " ",
+    "space-after-colon": " ",
+    "strip-spaces": "True"}
+
+CSSCombBearTest = verify_local_bear(CSSCombBear,
+                                    valid_files=(test_file1,),
+                                    invalid_files=(test_file2,
+                                                   test_file3, test_file4))
+
+
+CSSCombBearConfigFileTest = verify_local_bear(
+    CSSCombBear,
+    valid_files=(test_file5,),
+    invalid_files=(test_file2, test_file3),
+    settings={"csscomb_json": csscombconfig})
+
+
+CSSCombBearCoafileTest = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(),
+    valid_files=(test_file5,),
+    settings=settings)
+
+
+CSSCombBearCoafileTest2 = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(test_file2, test_file3, test_file4),
+    valid_files=(),
+    settings=settings)
+
+
+CSSCombBearSemicolonTest = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(test_file_always_semicolon,),
+    valid_files=(),
+    settings={"always-semicolon": "True"})
+
+
+CSSCombBearSpaceBefore = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(),
+    valid_files=(test_file1,),
+    settings={"space-before-colon": " "})
+
+
+CSSCombBearSpaceAfter = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(test_file2, test_file4),
```

Comment:
ooh i see....... they are invalid because you set `' '` as default value of `space_before`, that shouldn't be affecting this test, if you want to test `space_after` make sure the test files only fail/work because of the space after the colon 

What i am saying is since you set `space_before=' '` your test file here should have one space before the colon